airflow/providers/amazon/aws/example_dags/example_s3.py

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.models.dag import DAG
+from airflow.providers.amazon.aws.operators.s3_bucket import S3CreateBucketOperator, S3DeleteBucketOperator
+from airflow.utils.dates import days_ago
+
+DAG_NAME = 'aws_s3_dag'
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': True,
+    'start_date': days_ago(2)
+}
+
+with DAG(
+    dag_id=DAG_NAME,
+    schedule_interval=None,
+    default_args=default_args
+) as dag:
+
+    create_bucket = S3CreateBucketOperator(
+        task_id='aws_s3_create',
+        bucket_name='test-airflow-12345',
+        region_name='us-east-1',
+    )
+
+    delete_bucket = S3DeleteBucketOperator(
```

Comment:
Since this will delete objects from the bucket, you should put an object in to it before deleting.